perun-oidc-server/src/main/java/cz/muni/ics/oidc/server/claims/sources/EntitlementSource.java

```diff
@@ -57,19 +57,14 @@
 
 	public EntitlementSource(ClaimSourceInitContext ctx) {
 		super(ctx);
+
 		this.forwardedEntitlements = ClaimUtils.fillStringPropertyOrNoVal(FORWARDED_ENTITLEMENTS, ctx);
```

Comment:
Try to think if the `ClaimUtils.fillStringPropertyOrNoVal` method wouldn't be better dropped, and the `ClaimUtils.fillStringPropertyOrDefaultVal` should be instead, providing `null` as the default value.